src/gui/options.ui

```diff
@@ -719,6 +719,70 @@
+                 </property>
+                 <property name="checked">
+                  <bool>false</bool>
+                 </property>
+                 <layout class="QVBoxLayout" name="verticalLayout_17">
+                  <item>
+                   <widget class="QCheckBox" name="deleteCancelledTorrentBox">
+                    <property name="toolTip">
+                     <string>Delete also files for torrent, whose addition was cancelled</string>
+                    </property>
+                    <property name="text">
+                     <string>Also when addition is cancelled</string>
+                    </property>
+                   </widget>
+                  </item>
+                  <item>
+                   <layout class="QHBoxLayout" name="horizontalLayout_11">
+                    <item>
+                     <widget class="QLabel" name="deleteTorrentWarningIcon">
+                      <property name="sizePolicy">
+                       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
+                        <horstretch>0</horstretch>
+                        <verstretch>0</verstretch>
+                       </sizepolicy>
+                      </property>
+                      <property name="text">
+                       <string>I</string>
+                      </property>
+                     </widget>
+                    </item>
+                    <item>
+                     <widget class="QLabel" name="deleteTorrentWarningLabel">
+                      <property name="sizePolicy">
+                       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
+                        <horstretch>0</horstretch>
+                        <verstretch>0</verstretch>
+                       </sizepolicy>
+                      </property>
+                      <property name="toolTip">
+                       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;By enabling these options, you can&lt;span style=&quot; font-weight:600;&quot;&gt; irrevocably lose&lt;/span&gt; your .torrent files!&lt;/p&gt;&lt;p&gt;When these options are enabled, qBittorent will &lt;span style=&quot; font-weight:600;&quot;&gt;delete&lt;/span&gt; .torrent files after they were successfully (the first option) or not (the second option) added to its download queue. This will be applied &lt;span style=&quot; font-weight:600;&quot;&gt;not only&lt;/span&gt; to the files opened via quot;Add torrentquot; menu action but to those opened via &lt;span style=&quot; font-weight:600;&quot;&gt;file type association&lt;/span&gt; as well.&lt;/p&gt;&lt;p&gt;If you enable the second option (quot;Also when addition is cancelledquot;) the .torrent file &lt;span style=&quot; font-weight:600;&quot;&gt;will be deleted&lt;/span&gt; even if you press quot;&lt;span style=&quot; font-weight:600;&quot;&gt;Cancel&lt;/span&gt;quot; in the quot;Add torrentquot; dialog.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
`quot;` isn't replaced properly.

Using Qt Designer to input the text does produce the well-formed string for me...